baram/efs_manager.py

```diff
@@ -0,0 +1,79 @@
+import boto3
+import traceback
+
+from baram.log_manager import LogManager
+
+
+class EFSManager(object):
+    def __init__(self):
+        self.cli = boto3.client('efs')
+        self.logger = LogManager.get_logger()
+
+    def list_efss(self):
+        """
+        List one or more of your EFS.
+
+        :return:
+        """
+        return self.cli.describe_file_systems()['FileSystems']
+
+    def list_mount_targets(self, efs_id: str):
+        """
+        List mount targets of specific file system.
+
+        :param efs_id: FileSystemId
+        :return:
+        """
+        try:
+            return self.cli.describe_mount_targets(FileSystemId=efs_id)['MountTargets']
+        except:
+            traceback.print_exc()
+
+    def list_redundant_efss(self, redundant_sm_domain_ids: list = []):
+        """
+        Describe redundant file systems
+
+        :param redundant_sm_domain_ids: DomainId
+        :return: List of FileSystemId
+        """
+        # TODO: Need more cases (sagemaker case only)
+        redundant_efs_ids = [efs['FileSystemId'] for efs in self.list_efss()
+                             if efs['CreationToken'] not in redundant_sm_domain_ids
+                             and 'sagemaker' in efs['Tags'][0]['Value']]
+        return redundant_efs_ids
+
+    def delete_mount_targets(self, mount_target_id: str):
+        """
+        Delete mount targets via its id.
+
+        :param mount_target_id: MountTargetId
+        :return:
+        """
+        try:
+            self.cli.delete_mount_target(MountTargetId=mount_target_id)
+        except:
+            traceback.print_exc()
```

Comment:
trace back should print with line number.

Reply:
All `print_exc()` will be replaced to `format_exc()`. Thank you.